modules/apps/journal/journal-web/src/main/resources/META-INF/resources/edit_article.jsp

```diff
@@ -15,6 +15,54 @@ JournalEditArticleDisplayContext journalEditArticleDisplayContext = (JournalEdit
 journalEditArticleDisplayContext.setViewAttributes();
 %>
 
+<c:if test='<%= MultiSessionMessages.contains(renderRequest, "articleSavedAsDraft") %>'>
+	<c:if test="<%= article != null %>">
+		<liferay-util:buffer
+			var="alertMessage"
+		>
+			<liferay-util:buffer
+				var="articleLink"
+			>
+				<clay:link
+					cssClass="alert-link"
+					href='<%=
+						PortletURLBuilder.createRenderURL(
+							liferayPortletResponse
+						).setMVCRenderCommandName(
+							"/journal/edit_article"
+						).setRedirect(
+							currentURL
+						).setParameter(
+							"articleId", article.getArticleId()
+						).setParameter(
+							"backURLTitle", portletDisplay.getPortletDisplayName()
+						).setParameter(
+							"folderId", article.getFolderId()
+						).setParameter(
+							"groupId", article.getGroupId()
+						).setParameter(
+							"version", article.getVersion()
+						).buildString()
+					%>'
+					label="<%= article.getTitle(locale) %>"
+					translated="<%= false %>"
+				/>
+			</liferay-util:buffer>
+
+			<liferay-ui:message arguments="<%= articleLink %>" key="x-was-successfully-saved-as-draft" />
```

Comment:
@bakayattila, I'm unsure about this. When saving as draft, the user stays in the same page, why are we providing them with the link to the article? Wouldn't the name be enough in this case?

Reply:
I was copying the existing behavior. It might be dumb to provide a link to the item but it's consistent trough the use cases.

Reply:
I'm still unsure about it :smile:. Still, I concede if this makes it consistent